instrumentation/jedis/jedis-3.0/src/test/groovy/Jedis30ClientTest.groovy

```diff
@@ -76,29 +76,29 @@ class Jedis30ClientTest extends AgentTestRunner {
     assertTraces(2) {
       trace(0, 1) {
         span(0) {
-          name "SET"
+          name "SET foo ?"
           kind CLIENT
           attributes {
-            "${SemanticAttributes.DB_SYSTEM.key()}" "redis"
-            "${SemanticAttributes.DB_CONNECTION_STRING.key()}" "localhost:$port"
-            "${SemanticAttributes.DB_STATEMENT.key()}" "SET"
-            "${SemanticAttributes.NET_PEER_IP.key()}" "127.0.0.1"
-            "${SemanticAttributes.NET_PEER_NAME.key()}" "localhost"
-            "${SemanticAttributes.NET_PEER_PORT.key()}" port
+            "$SemanticAttributes.DB_SYSTEM.key" "redis"
+            "$SemanticAttributes.DB_CONNECTION_STRING.key" "localhost:$port"
+            "$SemanticAttributes.DB_STATEMENT.key" "SET foo ?"
+            "$SemanticAttributes.NET_PEER_IP.key" "127.0.0.1"
+            "$SemanticAttributes.NET_PEER_NAME.key" "localhost"
+            "$SemanticAttributes.NET_PEER_PORT.key" port
           }
         }
       }
       trace(1, 1) {
         span(0) {
-          name "GET"
+          name "GET foo"
```

Comment:
I am not sure about this. `foo` here is cache key, right? Thus its cardinality is high. Thousands and millions potentially. Semantic convention says that span name should be low-cardinality value. `select * from table_name` is Ok span name, but here in Redis having cache key as part of the name - I am not sure.

Reply:
Hmm, that's true. Lettuce 5.1 uses just the command name as span name, how about making all other Redis instrumentations do the same? I think it matches the spec:

> The span name SHOULD be set to a low cardinality value representing the statement executed on the database. It may be a stored procedure name (without arguments), SQL statement without variable arguments, operation name, etc.

Reply:
Yes, just command name seems good.

Reply:
Done.